Automattic/jetpack
# Search: hide 'Media' from 'Excluded Post Types'

Fixes #24681 

#### Changes proposed in this Pull Request:
The PR proposes changes that hide 'Media' from the 'Excluded Post Types', because we do not index 'Media' post type at all, meaning they are never returned from the search API.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Go to a site with Jetpack Search subscription
* Open Customizer `/wp-admin/customize.php`, find Jetpack Search section
* Ensure you do not see 'Media' from the 'Excluded Post Types'
* Open Customberg `/wp-admin/admin.php?page=jetpack-search-configure`
* Ensure you do not see 'Media' from the 'Excluded Post Types'

|Customizer|Customberg|
|-----------|-------------|
|![image](https://user-images.githubusercontent.com/1425433/175165430-de393870-6972-4cd8-9344-c669eb39436c.png)|![image](https://user-images.githubusercontent.com/1425433/175163012-ddb5cb04-cd5b-4177-ac29-85fd82e3cf04.png)|
